example/testapp/src/Makefile

```diff
@@ -1,7 +1,23 @@
 BIN_NAME=serv
+export GOPATH=$(realpath $(dir $(realpath $(lastword $(MAKEFILE_LIST)))).. )
+export GOROOT=$(realpath $(shell which go)/../..)
 
-bin: main.go
-	go build -o $(BIN_NAME)
+
+$(BIN_NAME): main.go
+	go build -i -o $(BIN_NAME)
```

Comment:
Added the `-i` to speed up compilation.  It apparently enables incremental builds so that only the changed files are recompiled.  In particular the sqlite package was very slow.